api/src/main/resources/workbench.yaml

```diff
@@ -550,6 +550,7 @@ paths:
       operationId: listClusters
       tags:
         - cluster
+        - compute
```

Comment:
may want to make this tag name more specific to billing lockout, something like billingRequired; we'll likely have compute-related endpoints in other contexts (e.g. administrative) which should not have the lockout restriction.

Reply:
+1 Also, are tags the only way to have custom attributes on paths? Or can we define something with more structure if we need it? It's somewhat worrying that we have to remember the first tag is the controller name, then this new tag pertains to billing, etc.

Also, please let me know if getting to OpenAPI 3 earlier would make your job easier here.